code/modules/reagents/oldchem/reagents/reagents_toxin.dm

```diff
@@ -26,6 +26,234 @@
+	else if (volume >= 15)
+		// bitten twice, die more quickly, muscle cramps make movement difficult. Call medics immediately.
+		// total damage: 4, human health 150 until crit, = 37.5 ticks, = 75s = 1m15s until death
+		M.adjustToxLoss(4)
+		M.confused = max(M.confused, 3)
+		M.eye_blurry = max(M.eye_blurry, 3)
+	else
+		// bitten once, die slowly. Easy to survive a single bite - just go to medbay.
+		// total damage: 2/tick, human health 150 until crit, = 75 ticks, = 150 seconds = 2.5 minutes to get to medbay.
+		M.adjustToxLoss(2) // same damage/tick as tabun cycle 0 to 60
+	..()
+	return
+
+// Terror Spider, White, Tranq
+
+/datum/reagent/terror_white_tranq
+	name = "White Spider tranquilizer"
+	id = "terror_white_tranq"
+	description = "A venom that incapacitites those who attack the White Death spider."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 0.1
+
+/datum/reagent/terror_white_tranq/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	// effects are similar to ketamine, aka the sleepy pen
+	if(current_cycle >= 3)
+		M.confused = max(M.confused, 5)
+	if(current_cycle >= 6)
+		M.eye_blurry = max(M.eye_blurry, 5)
+	if(current_cycle >= 10)
+		M.Paralyse(10)
+	..()
+	return
+
+// Terror Spider, White, Egg Venom
+
+/datum/reagent/terror_white_toxin
+	name = "White Spider venom"
+	id = "terror_white_toxin"
+	description = "A venom consisting of thousands of tiny spider eggs. When injected under the skin, they feed on living flesh and grow into new spiders."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 0.1
+	var/wdstage = 0
+	var/wdtreated = 0
+	var/awaymission_infection = 0
+	var/alternate_ending = 0
+
+/datum/reagent/terror_white_toxin/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	// I want there to be a (difficult, little-known) treatment for this, but nothing fits!
+	//	spaceacillin doesn't work because its an antibiotic not an antiparasite
+	//	charcoal doesn't work because its a toxin scrubber not an antiparasite
+	//	calomel doesn't work because its a purgative not an antiparasite
+	//	...
+	// What would make most sense, I suppose, is surgery... but... surgery is easy...
+	if (wdtreated)
+		if (prob(30))
+			to_chat(M,"<span class='notice'> You feel better, as your black flesh begins to heal.</span>")
+			M.reagents.remove_reagent("terror_white_toxin", 100)
+	else
+		if (holder.has_reagent("terror_white_antitoxin",1))
+			if (wdstage < 150)
+				wdtreated = 1
+				to_chat(M,"<span class='notice'> The antivenom burns in your veins!</span>")
+				M.adjustToxLoss(40)
+		else
+			wdstage += 1 // since charcoal, calomel, and dialysis can't purge us... only option left is our unique antidote
+	if (volume < 30)
+		volume += 10 // this is more than dialysis (-3) calomel (-5) and charcoal -1) combined could pump out in a tick.
+	else if (volume < 50)
+		volume += 1
+	else if (volume < 100)
+		volume += 0.1
```

Comment:
The cure formula is just mutagen+spaceacillin+venom. Not that hard. You already have venom in you. If you swallow a mutagen+spaceacillin pill, it will react inside you and cure you.

The spider eggs reagent organ insertion thing was added after I wrote this, and I did consider updating this to use a similar system, but I felt that was... suboptimal. White spiders can infect people faster than xenos can spawn and use huggers. So I felt it was OK for the cure to be potentially faster too. I did not want to require a xeno-like curing process when the infection process was also much faster.